recipes/pyagg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "30a1c5519696ff868104cfe7d33c8ec52e8698a8be6e21a3b6eb2d0c168f28aa" %}
+
+package:
+  name: pyagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/jwiggins/pyagg/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - cython
+    - freetype
+  run:
+    - python
```

Comment:
Yep, definitely needs `numpy x.x`. There is Cython with `cimport numpy` too.

Please add `numpy x.x` to `build` and `run` requirements.